frontend/tests/e2e/pageObjects/studentAdmin/institute/schoolContactsPage.js

```diff
@@ -0,0 +1,106 @@
+        this.contactLastName = Selector('#contactEditLastName');
+        this.contactEditEmail = Selector('#contactEditEmail');
+        this.contactEditPhoneNumber = Selector('#contactEditPhoneNumber');
+        this.contactEditPhoneExt = Selector('#contactEditPhoneExt');
+        this.contactEditAltPhoneNumber = Selector('#contactEditAltPhoneNumber');
+        this.contactEditAltPhoneExt = Selector('#contactEditAltPhoneExt');
+        this.contactEditStartDate = Selector('#editContactEffectiveDateTextField');
+        this.datePickerClickOne = Selector('.v-date-picker-header__value').child('div').child('button');
+        this.datePickerYear = Selector('.v-date-picker-years').find('li').withText('2022');
+        this.datePickerMonth = Selector('div').child('.v-date-picker-table').find('.v-btn__content').withText('JAN');
+        this.datePickerDay = Selector('div').child('.v-date-picker-table').find('.v-btn__content').withText('1');
+    }
+
+    async clickEditContactButton(){
+        await t.click(this.editContactButton);
+        log.info("Edit School Contact Button Clicked");
+    }
+
+    async editSchoolContact(){
+        await t.typeText(this.contactFirstName, 'Tony', { replace: true });
+        await t.typeText(this.contactLastName, 'Hawk', { replace: true });
+        await t.typeText(this.contactEditEmail, 'thawk@test.com', { replace: true });
+        await t.typeText(this.contactEditPhoneNumber, '2501234564', { replace: true });
+        await t.typeText(this.contactEditPhoneExt, '888', { replace: true });
+        await t.typeText(this.contactEditAltPhoneNumber, '2508854578', { replace: true });
+        await t.typeText(this.contactEditAltPhoneExt, '999', { replace: true });
+
+        await this.selectStartDate();
+
+        await t.click(this.saveContactButton);
+        log.info("School Contact Edit Complete");
+    }
+
+    async selectStartDate() {
+        await t.click(this.contactEditStartDate);
+
+        await t.click(this.datePickerClickOne()).wait(1000);
+        await t.click(this.datePickerClickOne()).wait(1000);
+        await t.click(this.datePickerYear()).wait(1000);
+        await t.click(this.datePickerMonth());
+        await t.click(this.datePickerDay());
+
+        log.info("Contact start date selected")
+    }
+
+    async verifySchoolContactDetails() {
+
+        await this.verifyContactName();
+        await this.verifyContactEmail();
+        await this.verifyContactPhoneNum();
+        await this.verifyContactPhoneNumExt();
+        await this.verifyContactAltPhoneNum();
+        await this.verifyContactAltPhoneNumExt();
+        await this.verifyContactStartDate();
+        log.info('Contact Verification Complete');
+    }
+    async verifyContactName(name){
+        await t.expect(this.principalContactName.withText('Tony Hawk').innerText).contains('Tony Hawk');
+        log.info("Contact Name Verified");
+    }
+    async verifyContactEmail(){
```

Comment:
Please follow the same pattern like you did for contactName. Allow a 'name/email/text' variable. Example, 

 async verifyContactEmail(email){
   await t.expect(this.principalContactEmail.withText(email).innerText).contains(email);
        log.info(`Contact Email ${email} Verified`);
 }

Reply:
All verify functions have been reverted to access variables. Value added to logging statements.